.changeset/sour-parents-look.md

```diff
@@ -0,0 +1,7 @@
+---
+'@compiled/utils': minor
+'@compiled/jest': minor
```

Comment:
Felt this primarily needs to be a minor to ensure it picks up the utils change as it's a cross-package.  Technically you should bump `@compiled/utils` and `@compiled/jest` at the same time, but no awareness of it.

Realistically I doubt it's used, so it's all just for show!

Reply:
Okay yeah that tracks. I don't think jest is needed to be minor because it's not a dependency of anything else though, so it could be patch + utils  / css stay as minor.